CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The intended audience of this file is for py42 consumers -- as such, changes that don't affect
 how a consumer would use the library (e.g. adding unit tests, updating documentation, etc) are not captured here.
 
+## 1.27.0 - 2023-08-04
```

Comment:
We removed APIs in this update, I understand they were already deprecated, but should this be a major version bump?

Reply:
My thought was no because the backend doesn't exist anyway, so updating versions or pinning an old version would have no effect.

Reply:
Honestly its closer to a bugfix chore imo

Reply:
But idk